org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/DocumentResource.xtend

```diff
@@ -44,9 +45,14 @@ class DocumentResource {
 	}
 
 	@PUT
-	def Response update(@PathParam("resourcePath") String resourcePath, String content, @Context HttpHeaders headers) {
-		documentProvider.save(resourcePath, content)
-		return status(NO_CONTENT).build
+	def Response update(@PathParam("resourcePath") String resourcePath, @QueryParam("rename") String rename, String content, @Context HttpHeaders headers) {
```

Comment:
not that it makes a big difference, but shouldn't `rename` be a boolean? (Or it could contain the target path.)

I was also wondering if this is the "proper RESTful way" of moving a resource, but there does not really seem to be a consensus on that… :disappointed:

Reply:
If rename is a boolean, I cannot make request with ?rename only, it would the always have to be ?rename=true leaving he question of what ?rename=false will be.  I'd prefer to keep it as String, making a comment that presence will make the difference and the content of the 'rename' parameter string is discarded